extensions/amp-story/0.1/amp-story.js

```diff
@@ -1004,6 +997,36 @@ export class AmpStory extends AMP.BaseElement {
   }
 
 
+  /** @private */
+  onCurrentPageIdUpdate_(pageId) {
+    this.setHistoryState_(pageId);
+  }
+
+
+  /**
+   * Save page id using history API
+   * @private
+   */
+  setHistoryState_(pageId) {
```

Comment:
Please make both the getter and setter consistent:

either

````
state = {};
setHistoryState_(state)
getHistoryState_() // returns {}
````

or

````
pageId = 'intro';
setHistoryStatePageId_(pageId)
getHistoryStatePageId_() // returns 'intro'
````

Reply:
option 2 done.